vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1975,23 +1994,12 @@ def _bookkeeping_sync(
         if envs.VLLM_COMPUTE_NANS_IN_LOGITS:
             num_nans_in_logits = self._get_nans_in_logits(logits)
 
-        # TODO(woosuk): The following loop can be slow since it iterates over
-        # the requests one by one. Optimize.
-        discard_sampled_tokens_req_indices = []
-        for i, req_id in enumerate(self.input_batch.req_ids):
-            req_state = self.requests[req_id]
-            seq_len = (req_state.num_computed_tokens +
-                       scheduler_output.num_scheduled_tokens[req_id])
-            if seq_len < req_state.num_tokens:
-                # Ignore the sampled token for partial prefills.
-                # Rewind the generator state as if the token was not sampled.
-                # This relies on cuda-specific torch-internal impl details
```

Comment:
@benchislett sorry was a bit late seeing this, but just wondering why these comments were removed. In particular the warning about cuda-specific torch internal details. These generator offsets aren't public APIs or applicable to all torch backend impls (e.g. CPU)